src/styles/blocks/header.scss

```diff
@@ -0,0 +1,99 @@
+* {
+  padding: 0;
+  margin: 0;
+  list-style: none;
+  box-sizing: border-box;
+}
+
+a {
+  text-decoration: none;
+  color: inherit;
+}
+
+html {
+  font-size: 16px;
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: 500;
```

Comment:
The font settings for the `html` element are usually set in a more general stylesheet file like `src/styles/index.scss`. If this is the main font style for the entire project, consider moving it there for better maintainability and to avoid duplication.